app/views/layouts/errors_and_contact.html.erb

```diff
@@ -11,7 +11,6 @@
 
   <body class="govuk-template__body js-enabled">
     <%= render '/layouts/errors_and_contact_header' %>
-    <%= render '/layouts/phase_banner' %>
```

Comment:
I think this removal is accidental as well?

Reply:
(I could re-add this myself, but I want to double-check I haven't misunderstood something!)

Reply:
Re-added this @kjdchapman